scripts/package.json

```diff
@@ -7,9 +7,13 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
+        "chalk": "^5.0.0",
+        "fs-extra": "^10.0.1",
```

Comment:
Is there any strong reason to add a new package for something that nodejs has built-in? 
Do you need to do something that is not possible to do with nodejs built-in [fs](https://nodejs.org/api/fs.html) ?

Reply:
`fs-extra` has some really nice functions that are not in base `fs`, like recursive copy of folders. I am pretty sure that I will be handy for more utilities scripts that will come in future, also it is already in `node_modules` because some of our deps using this underhood.

Reply:
I can definitely write some utility script that will use plain `fs.copy` and some reading of structure of folder, but for these "development, DX only things" I nearly always prefer to use npm package, since code quality will be in most of cases much better, with test coverage... and it will save lot of time.